spec-tests/src/main/java/org/candlepin/spec/bootstrap/data/builder/Contents.java

```diff
@@ -78,7 +78,8 @@ public static ContentDTO copy(ContentDTO source) {
      *  a ContentDTO instance with the required fields populated with semi-random values
      */
     public static ContentDTO random() {
-        String cid = StringUtil.random(18, StringUtil.CHARSET_NUMERIC);
+        // prefix is needed because this id is used in a OID. BC does not allow leading zeros in a segment
+        String cid = StringUtil.random("1", 17, StringUtil.CHARSET_NUMERIC);
```

Comment:
This works, and is probably acceptable, but you could have just as easily just removed 0s from the character set:

`String cid = StringUtil.random(18, "123456789");`